clients/java/src/main/resources/templates/typeInfoAnnotation.mustache

```diff
@@ -0,0 +1,18 @@
+{{#jackson}}
+
+{{#discriminator.mappedModels}}
+{{#-first}}
+@JsonSubTypes({
```

Comment:
❓ JsonSubTypes is from and used by Jackson, right? Why do we also have `JsonbTypeInfo` from Jakarta, will that be or is it already relevant for bean validation?